tests/src/test/java/util/ItUtils.java

```diff
@@ -464,22 +467,15 @@ public static Date toDate(String sDate) {
   }
 
   public static Date toDatetime(String sDate) {
-    try {
-      SimpleDateFormat sdf = new SimpleDateFormat("yyyy-MM-dd'T'HH:mm:ssZ");
-      return sdf.parse(sDate);
-    } catch (ParseException e) {
-      throw new RuntimeException(e);
-    }
+    return Date.from(OffsetDateTime.parse(sDate, ISO_DATE_TIME).toInstant());
```

Comment:
It would have been more readable to split this commit into 2 : 
- One about refactoring to sanitze code
- One about using new format